packages/compass-query-bar/src/stores/ai-query-reducer.ts

```diff
@@ -112,6 +112,12 @@ export const runAIQuery = (
   Promise<void>,
   AIQueryStartedAction | AIQueryFailedAction | AIQuerySucceededAction
 > => {
+  track('AI Prompt Submitted', () => ({
+    editor_view_type: 'find',
+    // One token generally corresponds to ~4 characters of text for common English text.
+    num_tokens: userInput.length / 4,
```

Comment:
I'd suggest we keep this named something like user_input_length  as it doesn't necessarily indicate the amount of tokens.

Reply:
The "compass tracking plan" specifies `num_tokens`, but I like your suggestion here. Let's use characters length and discuss it with Julia when she is back.